site/content/blog/2021-08-05-v10-v13.md

```diff
@@ -0,0 +1,45 @@
+
+The Rust community evolves quickly. The async runtime ecosystem, even faster. Not long ago, the Tokio team released [Tokio 1.0](https://tokio.rs/blog/2020-12-tokio-1-0). This was a big win for the Rust ecosystem overall. Since then, they've continued to crank out new 1.x releases, along with new support libraries, io_uring support (love it), as well as a new server framework called [Axum](https://github.com/tokio-rs/axum) ([blog post](https://tokio.rs/blog/2021-07-announcing-axum)).
+
+All that to say, I decided to cut Trunk over to the Tokio ecosystem, and while I was in the middle finishing up that work, I found myself wishing that we had a nice, simple, ergonomic web framework for Tokio ... and then literally the very next day Axum was released. This couldn't have come at a better time. Axum is built on Hyper & Tower, so it has direct interoperability with that entire ecosystem. This made the switch rather simple on the Trunk side.
+
+I have to say, I'm quite stoked. Very happy with how this turned out for Trunk, and happy to see this new option come about for the Rust ecosystem. The wins here are obvious, but the biggest win is that we have a greater stability guarantee overall, and the success of the Tokio ecosystem to draw from.
+
+## Other Features & Fixes
+Lots of big wins so far, but these last few releases have also seen many other smaller features and fixes. Let's take a look.
+
+### `indicatif`
+After sinking more time than I care to admit, and finding myself frustrated rather often with `indicatif`, I decided to completely remove it from Trunk. Now, to be sure, `indicatif` is great, and its maintainer is quite literally a hero among the greater software community, but the difficulties I ran into here seemed to be more closely related to async interop.
+
+Without going into too many details, deadlocks from progress bars, consoles not clearing properly, inflexibility of design, all of these things were causing me to spend more time on the console than spending time building other **real** features for Trunk. So, I decided to instead just use the good ol' [tracing](https://github.com/tokio-rs/tracing) crate, and now things are quite stable.
+
+Truth be told, I would love to circle back at some point in the future and add a nicer console UI to Trunk. This is currently fairly low priority for me, as this is purely aesthetic. If you are passionate about such things, drop by our community discord and let us know!
+
+There are many other features and fixes of note. Have a look at the [changelog](https://github.com/thedodd/trunk/blob/master/CHANGELOG.md) for more details.
+
+## Conclusion
+All in all, some great progress for Trunk. As far as Trunk's future, the plugin system and exploring patterns around code splitting and HMR and the things I'm most excited about.
```

Comment:
Sorry, wasn't patient enough to wait until you actually publish it :)
```suggestion
All in all, some great progress for Trunk. As far as Trunk's future, the plugin system and exploring patterns around code splitting and HMR are the things I'm most excited about.
```

Reply:
Nice! Good eye.